btree/btree.c

```diff
@@ -1908,10 +1925,28 @@ static int64_t tree_get(struct node_op *op, struct segaddr *addr, int nxt)
 static int64_t tree_create(struct node_op *op, struct m0_btree_type *tt,
 			   int rootshift, struct m0_be_tx *tx, int nxt)
 {
-	return segops->so_tree_create(op, tt, rootshift, tx, nxt);
+	struct td *tree;
```

Comment:
This code is under `#if 0`. are we not going to need this code?

Reply:
This code is used in UT only as this code considers that we dont have space allocated for root node, and then it goes ahead an creates a tree with empty root node. This code has been put under if #0 by shashank as there is a bit dilemma whether to keep it or not.